src/AutoFilterer/Types/PaginationFilterBase.cs

```diff
@@ -9,21 +9,21 @@ namespace AutoFilterer.Types;
 public class PaginationFilterBase : OrderableFilterBase, IOrderablePaginationFilter
 {
     [IgnoreFilter]
-    public virtual int Page { get; set; } = 1;
+    public virtual int? Page { get; set; } = 1;
 
     [IgnoreFilter]
-    public virtual int PerPage { get; set; } = 10;
+    public virtual int? PerPage { get; set; } = 10;
 
     public override IQueryable<TEntity> ApplyFilterTo<TEntity>(IQueryable<TEntity> query)
     {
         if (query is IOrderedQueryable<TEntity> ordered)
             return this.ApplyFilterTo(ordered);
 
-        return base.ApplyFilterTo(query).ToPaged(Page, PerPage);
+        return base.ApplyFilterTo(query).ToPaged(Page ?? 1, PerPage ?? 10);
```

Comment:
When Page or PrePage is null, consider skipping pagination.

Reply:
This should not be modified, it is required to pass in paging parameters here. What's your opinion.

Reply:
We have `ApplyFilterWithoutPagination()` method to ignore pagination. It's not client-controlled since skipping pagination may cause big performance problems and a bottleneck in the main application. When required, calling `query.ApplyFilterWithoutPagination(filter)` method in the code is a good approach. Makes the developer know about no-filtering instead of being managed by the client. But it depends on case, but this one is more common